provider/storage_benchmark.py

```diff
@@ -51,26 +51,48 @@ class StorageBenchmark(object):
                      CURL_DOWNLOAD: 'curl -o {0} {1}'}
     unpack_cmds = {TAR_UNPACK: '_tar_unpack_file'}
 
-    def __init__(self, os_type, session, name):
+    def __init__(self, os_type, vm, name):
         """
-        :param session: session connected guest
+        :param vm: vm object
+        :type vm: qemu_vm.VM object
         :param name: the name of benchmark
         :type name: str
         """
-        self.session = session
+        self.vm = vm
         self.name = name
         self.os_type = os_type
         self.env_files = []
+        self.session = self._create_session()
```

Comment:
call `self._create_session()` is enough, it will create `self.session`

Reply:
Thanks, have updated.